src/vm/instruction_set.md

```diff
@@ -647,6 +679,363 @@ Panic if:
 
 `$err` is cleared.
 
+### WDCM: 128-bit integer comparison
+
+|             |                                                                                       |
+|-------------|---------------------------------------------------------------------------------------|
+| Description | Compare or examine two 128-bit integers using selected mode                           |
+| Operation   | `b = mem[$rB,16]`<br>;`c = indirect?mem[$rC,16]:$rC;`<br>`$rA = cmp_op(b,c);`         |
```

Comment:
I don't think this is rendering correctly

Reply:
Fixed.